### PR TITLE
Fixed typos and improved punctuation consistency in README

### PR DESCRIPTION
**Pull request by muesli**
Just a couple of nitpicky grammar, typo and punctuation fixes for the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ makepkg -si
 
 Contributors are always welcome!
 
-If you plan to make any large changes or changes that may not be 100% agreed
+If you plan to make larger changes or changes that may not be 100% agreed
```
Comment (Morganamilo) [inline README.md]: I prefer the old wording. Is there anything wrong with it?

File: README.md [hunk 2]
```diff
@@ -107,11 +107,12 @@ Currently yay Depends on:
   `yay -{OPERATION} --repo`
 
 ### `Out Of Date AUR Packages` message is displayed, why doesn't `yay` update them?
-  This means the package has been flagged out of date on the AUR but maintainer has not updated the `PKGBUILD` yet.
+  This means the package has been flagged out of date on the AUR, but its
+  maintainer has not updated the `PKGBUILD` yet.
```
[the remark this answers is not in the document]
Comment (muesli) [reply README.md]: Sounds fine to me, both work. Will amend.

File: README.md (hunk 1 above)
Comment (muesli) [reply README.md]: The combination of "any" with the plural "changes" sounds a bit weird to some people, even though it's not technically incorrect as far as I know. I believe this change makes it sound a bit more natural (native speakers: please correct me here).

Comment (Morganamilo) [reply README.md]: Well larger means bigger than something. So to me when you say larger without comparing it to anything it does not quite make sense.

Comment (muesli) [reply README.md]: Then we should probably remove the word "large" altogether, as you can't really quantify code changes that way (where does large begin?). How about "If you plan to make more involved/complex/elaborate/intricate changes..."?

Comment (Morganamilo) [reply README.md]: You could say the same thing about all of these. What makes a change complex? Yes large is subjective but that's fine. And a change does not have to be complex to merit an issue anyway. You could be tearing out a large piece of the code base in order to make it less complex. Basically what #585 is doing right now.

Comment (Morganamilo) [reply README.md]: @polygamma you're stalking the repo right now. Got any opinions with your German-English?

Comment (polygamma) [reply README.md]: my 2 cents: `If you plan to make any large changes` is fine

Comment (muesli) [reply README.md]: As mentioned I'm not a native speaker either, but I have spend half my life in english speaking countries. The word 'any' with an innumerable plural object simply sounds a bit awkward.

A 'large change' by definition stands in contrast to a smaller change, hence why I would use the comparative "larger" here.

"Edits from maintainers" are enabled, so you do you here.

Comment (Jguer) [reply README.md]: Both of them sound fine. Personally I prefer `any large changes` even if it's more Americanised

Comment (Morganamilo) [reply README.md]: @muesli There's three of us in agreement on keeping it. It would be much easier if you edited your own commits than asking us to do it.

**Comment by Morganamilo** [comment]
Didn't realize you made the change already. (rebasing a PR does not give email notifications). Anyway thanks for the changes.

**Pull request merged**